infra/k8s/sidecar.yaml

```diff
@@ -19,6 +19,21 @@ spec:
       hostPID: true
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
+      initContainers:
+      - name: iproute-add
+        image: busybox
+        securityContext:
+          privileged: true
+        command:
+        - sh
+        - -ac
+        - >
+          sleep 20 &&
+          export GW=$(ip route | grep cni0 | awk '{print $7}') &&
+          echo $GW &&
+          ip route &&
+          ip route add 100.64.0.0/16 via $GW &&
+          ip route || true
```

Comment:
It'd be nice for this command to detect if the `route` is already there, so that we can restart the `sidecar` daemonset.

Reply:
Might not need to use the dummy container or sleep here, see https://github.com/ipfs/testground/issues/416#issuecomment-579060016

Reply:
I will try to use `--networking flannel` and see if the interface is created. If `cni0` is indeed created, then we don't need a dummy container or sleep, otherwise we still need them until we come up with another solution.

Reply:
@gmasgras when starting `kops` with `--networking flannel`, we still need to create a pod on every host, so that we get the `cni0` interface and then use it to add the DNS route (100.64.0.0/16).

We should get rid of the sleep, and adding pooling in this cmd, so that it is more consistent, and not time-based.